src/main/java/org/generationcp/middleware/api/germplasm/search/GermplasmSearchService.java

```diff
@@ -13,7 +14,7 @@
 
 	long countSearchGermplasm(GermplasmSearchRequest germplasmSearchRequest, String programUUID);
 
-	List<UserDefinedField> getGermplasmAttributeTypes(GermplasmSearchRequest germplasmSearchRequest);
+	List<CVTerm> getGermplasmAttributeTypes(GermplasmSearchRequest germplasmSearchRequest);
```

Comment:
@cuenyad avoid exposing CVTerm 
rename by getGermplasmAttributeVariables

Reply:
@clarysabel 
OK, I will return Variable instead and I modify the query to retrieve the alias also.